### PR TITLE
Update AboutUs.md

### PR DESCRIPTION
**Pull request by Gordon25**
- [x] include links to email
- [x] reformat personal particulars row

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -52,8 +57,9 @@ We are a team based in the [School of Computing, National University of Singapor
 
 <img src="images/giang.png" width="200px">
```
Comment (chunweii) [inline docs/AboutUs.md:58]: Should be pcgiang.png

Comment (chunweii) [reply docs/AboutUs.md:58]: Update the name of Giang's image inside the images folder too.

**Review by chunweii — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (Gordon25) [reply docs/AboutUs.md]: Updated changes

Comment (chunweii) [reply docs/AboutUs.md]: No, **do not change the github account link**!

**Review by chunweii — CHANGES_REQUESTED** [review]
Giang's github link should not have `.png`

**Review by chunweii — APPROVED** [review]
~~LGTM~~

Your name is repeated twice in the heading. Did not catch that initially. Please change

**Pull request merged**